src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/servingclaims/ServingClaimsReportTest.java

```diff
@@ -57,7 +58,7 @@ public void setUp() {
         bfActionType.setNotes("Test Notes One");
         bfActionTypeItem.setId("0011");
         bfActionTypeItem.setValue(bfActionType);
-        caseData.setBfActions(new ArrayList<>(Arrays.asList(bfActionTypeItem)));
+        caseData.setBfActions(new ArrayList<>(Collections.singletonList(bfActionTypeItem)));
```

Comment:
It's nice to see someone trying to get rid of these daft `new ArrayList<>(Arrays.asList` we have everywhere.
But this would probably a better alternative:

`caseData.setBfActions(List.of(bfActionTypeItem));`

Reply:
Refactored